pkg/nodeoperations/controller.go

```diff
@@ -36,9 +37,20 @@ import (
 )
 
 const (
-	nodeRemovalTaintKey    = "node.dell.com/drain"
-	nodeRemovalTaintValue  = "drain"
-	nodeRemovalTaintEffect = "NoSchedule"
```

Comment:
Why don't you check for `NoSchedule` anymore?

Reply:
As I understand effect it's only for kubernetes. Are there requirement that effect need to check? flex-operator skips it.